src/components/views/rooms/RoomListPanel/RoomListHeaderView.tsx

```diff
@@ -70,8 +70,9 @@ export function RoomListHeaderView(): JSX.Element {
                             <ComposeMenu vm={vm} />
                         ) : (
                             <IconButton
-                                aria-label={_t("action|start_chat")}
+                                aria-label={_t("action|new_conversation")}
                                 onClick={(e) => vm.createChatRoom(e.nativeEvent)}
+                                tooltip={_t("action|new_conversation")}
                             >
                                 <ComposeIcon color="var(--cpd-color-icon-secondary)" />
```

Comment:
Two things that could be cleaned up here, while you're at it: The `tooltip` prop already acts as an accessible label, so `aria-label` is redundant. Also we should `aria-hidden` the icon itself since we don't intend to label it separately from the button.

```suggestion
                                onClick={(e) => vm.createChatRoom(e.nativeEvent)}
                                tooltip={_t("action|new_conversation")}
                            >
                                <ComposeIcon color="var(--cpd-color-icon-secondary)" aria-hidden />
```

Perhaps we could eventually make this less error-prone with some adjustments to the IconButton API.